### DISCUSSION TITLE
dpgen init_reaction  dataset_size

### DISCUSSION THREAD

**Discussion by nnyyxx-bit** [tongzhugroup/mddatasetbuilder#190] [Q&A]
How should the dataset_size in init_reaction be set up. (Ammonia Combustion System)
![屏幕截图 2023-11-07 220216](https://github.com/tongzhugroup/mddatasetbuilder/assets/144452397/e796a88f-883a-414e-a040-d9097ce94867)

**Comment by njzjz** [comment]
The structures sampled from ReaxFF are inaccurate, so it is not valuable to collect too much data in the initial stage.

**Reply by nnyyxx-bit** [reply]
How about dataset_size=50?

**Reply by njzjz** [reply]
You can try it and see whether the first simulation runs as expected or crashes.

**Reply by nnyyxx-bit** [reply]
Does the first simulation mean running a ReaxFF MD or running a simulation with the first trained model?
